README.md

```diff
@@ -175,7 +174,23 @@ make test-integration    # Run the integration tests
 ```
 
 
-## Licence
+Support and Migration
+---------------------
+
+Pa11y CI major versions are normally supported for 6 months after their last minor release. This means that patch-level changes will be added and bugs will be fixed. The table below outlines the end-of-support dates for major versions, and the last minor release for that version.
+
+We also maintain a [migration guide](MIGRATION.md) to help you migrate.
+
+| :grey_question: | Major Version | Last Minor Release | Node.js Versions | Support End Date |
+| :-------------- | :------------ | :----------------- | :--------------- | :--------------- |
+| :heart:         | 5             | N/A                | 8+               | N/A              |
+| :hourglass:     | 1             | 1.3                | 4+               | 2018-04-18       |
```

Comment:
I'd change the date and last minor release to TBC or something for now, just in case this takes a while to go out?

Reply:
I aspire to not let it take that long ;) (Plus CI is pretty rare to release tbh)